example/src/Bar.purs

```diff
@@ -142,12 +142,12 @@ genInp = F.for (Arr.range 0 10) \i → do
   four ← random <#> ((_ + 0.3) >>> U.precise 2.0)
   pure {label, one, two, three, four}
 
-chart ∷ ∀ e. Eff (random ∷ RANDOM, dom ∷ DOM, echarts ∷ ET.ECHARTS, err ∷ EXCEPTION|e) Unit
-chart = do
+chart ∷ ∀ e. Maybe EC.Theme → Eff (random ∷ RANDOM, dom ∷ DOM, echarts ∷ ET.ECHARTS, err ∷ EXCEPTION|e) Unit

```

Comment:
Could you please leave `chart` signature as is and make it something like `chart = chart' Nothing`?